HomeWork_05/HW_05_Task_02/ArrayModifier.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace HW_05_Task_02
+{
+    public class ArrayModifier
+    {
+        public int[] ArrayCreation(int arrayLength)
```

Comment:
If you have one method such as "ModifyArray" - why do not rename this method to `CreateArray` of vise versa